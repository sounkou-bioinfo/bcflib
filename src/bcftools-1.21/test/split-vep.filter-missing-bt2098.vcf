##fileformat=VCFv4.1
##contig=<ID=chr1>
##INFO=<ID=CSQ,Number=.,Type=String,Description="Consequence annotations from Ensembl VEP. Format: Allele|Consequence|IMPACT|SYMBOL|Gene|Feature_type|Feature|BIOTYPE|EXON|INTRON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|Codons|Existing_variation|ALLELE_NUM|DISTANCE|STRAND|FLAGS|PICK|VARIANT_CLASS|SYMBOL_SOURCE|HGNC_ID|CANONICAL|TSL|CCDS|ENSP|SWISSPROT|TREMBL|UNIPARC|UNIPROT_ISOFORM|RefSeq|SOURCE|GENE_PHENO|SIFT|PolyPhen|DOMAINS|HGVS_OFFSET|AF|AFR_AF|AMR_AF|EAS_AF|EUR_AF|SAS_AF|gnomADe_AF|gnomADe_AFR_AF|gnomADe_AMR_AF|gnomADe_ASJ_AF|gnomADe_EAS_AF|gnomADe_FIN_AF|gnomADe_NFE_AF|gnomADe_OTH_AF|gnomADe_SAS_AF|gnomADg_AF|gnomADg_AFR_AF|gnomADg_AMI_AF|gnomADg_AMR_AF|gnomADg_ASJ_AF|gnomADg_EAS_AF|gnomADg_FIN_AF|gnomADg_MID_AF|gnomADg_NFE_AF|gnomADg_OTH_AF|gnomADg_SAS_AF|MAX_AF|MAX_AF_POPS|CLIN_SIG|SOMATIC|PHENO|PUBMED|MOTIF_NAME|MOTIF_POS|HIGH_INF_POS|MOTIF_SCORE_CHANGE|TRANSCRIPTION_FACTORS|CADD_PHRED|CADD_RAW|CADD_PHRED|CADD_RAW|BayesDel_noAF_pred|BayesDel_noAF_score|ClinPred_pred|ClinPred_score|MetaRNN_pred|MetaRNN_score|Polyphen2_HDIV_pred|REVEL_score|SIFT4G_pred|SIFT_pred|clinvar_OMIM_id|clinvar_clnsig|clinvar_id|clinvar_review|clinvar_trait|clinvar_var_source|gnomad_genomes|gnomad_genomes_AF|gnomad_genomes_AC|gnomad_genomes_AN|gnomad_genomes_nhomalt">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
chr1	97630165	.	T	C	.	.	CSQ=C|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00001141396|enhancer||||||||||rs1480440198|1||||1|SNV|||||||||||||||||||||||||||||||||||||||||||||||||||||||8.208|0.673305|8.208|0.673305|||||||||||||||||||||,C|intergenic_variant|MODIFIER|||||||||||||||rs1480440198|1|||||SNV|||||||||||||||||||||||||||||||||||||||||||||||||||||||8.208|0.673305|8.208|0.673305|||||||||||||||||||||
chr1	116938163	.	G	C	.	.	CSQ=C|upstream_gene_variant|MODIFIER|AARD|ENSG00000205002|Transcript|ENST00000378279|protein_coding||||||||||rs1554616421|1|44|1||1|SNV|HGNC|HGNC:33842|YES|1|CCDS34935.1|ENSP00000367528|Q4LEZ3.97||UPI0000419472||NM_001025357.3||||||||||||||||||||||0.0002334|0.0001014|0|0.0001398|0|0.001988|0.0003349|0|0.0001708|0|0.001303|0.001988|gnomADg_EAS||||||||||7.584|0.607429|7.584|0.607429|||||||||||||||||rs1554616421|0.015909|6316|397018|19,C|upstream_gene_variant|MODIFIER|AARD|ENSG00000205002|Transcript|ENST00000523536|protein_coding_CDS_not_defined||||||||||rs1554616421|1|383|1|||SNV|HGNC|HGNC:33842||3|||||||||||||||||||||||||||||0.0002334|0.0001014|0|0.0001398|0|0.001988|0.0003349|0|0.0001708|0|0.001303|0.001988|gnomADg_EAS||||||||||7.584|0.607429|7.584|0.607429|||||||||||||||||rs1554616421|0.015909|6316|397018|19,C|regulatory_region_variant|MODIFIER|||RegulatoryFeature|ENSR00000229087|promoter||||||||||rs1554616421|1|||||SNV|||||||||||||||||||||||||||||||||0.0002334|0.0001014|0|0.0001398|0|0.001988|0.0003349|0|0.0001708|0|0.001303|0.001988|gnomADg_EAS||||||||||7.584|0.607429|7.584|0.607429|||||||||||||||||rs1554616421|0.015909|6316|397018|19,C|TF_binding_site_variant|MODIFIER|||MotifFeature|ENSM00198861829|||||||||||rs1554616421|1||1|||SNV|||||||||||||||||||||||||||||||||0.0002334|0.0001014|0|0.0001398|0|0.001988|0.0003349|0|0.0001708|0|0.001303|0.001988|gnomADg_EAS|||||ENSPFM0086|10|Y|-0.058|EGR1&EGR2&EGR4&EGR3|7.584|0.607429|7.584|0.607429|||||||||||||||||rs1554616421|0.015909|6316|397018|19
chr1	132003572	.	A	G	.	.	CSQ=G|intron_variant|MODIFIER|EFR3A|ENSG00000132294|Transcript|ENST00000254624|protein_coding||22/22|ENST00000254624.10:c.2360+287A>G||||||||1||1||1|SNV|HGNC|HGNC:28970|YES|1|CCDS34942.2|ENSP00000254624|Q14156.161||UPI00001C1E8F|Q14156-1|NM_001323558.2&NM_015137.6||||||||||||||||||||||||||||||||||||||||||||5.137|0.374079|5.137|0.374079|||||||||||||||||||||,G|intron_variant|MODIFIER|EFR3A|ENSG00000132294|Transcript|ENST00000519656|protein_coding||22/22|ENST00000519656.1:c.2252+287A>G||||||||1||1|||SNV|HGNC|HGNC:28970||1|CCDS83328.1|ENSP00000428086|Q14156.161||UPI0000D62503|Q14156-2|NM_001323555.2&NM_001323554.2&NM_001323553.2&NM_001323557.2&NM_001323556.2||||||||||||||||||||||||||||||||||||||||||||5.137|0.374079|5.137|0.374079|||||||||||||||||||||,G|downstream_gene_variant|MODIFIER|EFR3A|ENSG00000132294|Transcript|ENST00000519920|protein_coding_CDS_not_defined|||||||||||1|2616|1|||SNV|HGNC|HGNC:28970||5|||||||||||||||||||||||||||||||||||||||||||||||||||5.137|0.374079|5.137|0.374079|||||||||||||||||||||,G|intron_variant&non_coding_transcript_variant|MODIFIER|EFR3A|ENSG00000132294|Transcript|ENST00000521940|protein_coding_CDS_not_defined||4/4|ENST00000521940.1:n.447+287A>G||||||||1||1|||SNV|HGNC|HGNC:28970||3|||||||||||||||||||||||||||||||||||||||||||||||||||5.137|0.374079|5.137|0.374079|||||||||||||||||||||,G|intron_variant&non_coding_transcript_variant|MODIFIER|EFR3A|ENSG00000132294|Transcript|ENST00000523074|retained_intron||5/5|ENST00000523074.5:n.541+287A>G||||||||1||1|||SNV|HGNC|HGNC:28970||3|||||||||||||||||||||||||||||||||||||||||||||||||||5.137|0.374079|5.137|0.374079|||||||||||||||||||||,G|intron_variant|MODIFIER|EFR3A|ENSG00000132294|Transcript|ENST00000637848|protein_coding||22/22|ENST00000637848.1:c.2441+287A>G||||||||1||1|||SNV|HGNC|HGNC:28970||5||ENSP00000490312||A0A1B0GUZ7.35|UPI0007E52B5F||||||||||||||||||||||||||||||||||||||||||||||5.137|0.374079|5.137|0.374079|||||||||||||||||||||
