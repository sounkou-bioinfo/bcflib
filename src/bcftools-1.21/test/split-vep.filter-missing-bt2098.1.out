chr1 97630165 	 .,.
chr1 116938163 	 0.015909,0.015909,0.015909,0.015909
chr1 132003572 	 .,.,.,.,.,.
